includes/theme-support/class-llms-theme-support.php

```diff
@@ -29,6 +29,184 @@ public function __construct() {
 		add_action( 'after_setup_theme', array( $this, 'includes' ) );
 	}
 
+	/**
+	 * Retrieve formatted inline CSS for a give list of selectors and rules
+	 *
+	 * @since [version]
+	 *
+	 * @param string[] $selectors       Array of CSS selectors.
+	 * @param string[] $rules           Associative array of CSS rules and properties. For example: `array( 'color' => '#fff' )`.
+	 * @param string   $selector_prefix A css selector to prefix each item in $selectors with.
+	 * @return string
+	 */
+	public static function get_css( $selectors, $rules, $selector_prefix = '' ) {
+
+		// Convert the $rules array to a list of CSS strings.
+		$rules_list = array();
+		foreach ( $rules as $prop => $val ) {
+			$rules_list[] = sprintf( '%1$s: %2$s;', $prop, $val );
```

Comment:
I propose the following to allow using CSS rule fall backs/vendor specific e.g.
```css
background-image: -webkit-radial-gradient(center,ellipse,var( --global--color-secondary ) 0,var( --global--color-secondary ) 40%,#fafafa 45%);
background-image: radial-gradient(ellipse at center,var( --global--color-secondary ) 0,var( --global--color-secondary ) 40%,#fafafa 45%);
```
```suggestion
			$val = is_array( $val ) ? $val : array( $val );
			foreach ( $val as $value ) {
				$rules_list[] = sprintf( '%1$s: %2$s;', $prop, $value );
			}
```